Pod/Classes/ResizingTextView.swift

```diff
@@ -14,6 +14,16 @@ open class ResizingTextView: UITextView {
     
     private var heightConstraint: NSLayoutConstraint!
     
+    private var maxHeight: CGFloat?
+    
+    public var resizeForNumberOfLines: Int = 0 {
+        didSet {
+            guard resizeForNumberOfLines > 0 else { return (maxHeight = nil) }
+            let padding = (textContainer.lineFragmentPadding * 2) + textContainerInset.top + textContainerInset.bottom
+            maxHeight = (font!.lineHeight * CGFloat(resizeForNumberOfLines)) + padding
```

Comment:
Setting maxHeight here in the `didSet` doesn't account for changes to the `font`, `lineFragmentPadding`, and container insets that might occur AFTER the max lines is set.

We could fix this by doing this calculation in the `didUpdateText(_:)` function. To be a little bit more performant, alternatively, we could listen for changes to the font, padding, and insets on the text container and then cache the max/min height, but I think that's overkill.

I think we should just do this calculation as part of `didUpdateText(_:)`.  I would say to create a private function `calculateHeight(for resizableHeight: ResizableHeight) -> CGFloat?` and call it in `didUpdateText(_:)`. And then you can kill the `maxHeight` variable.